packages/commonwealth/server/controllers/server_communities_methods/update_community.ts

```diff
@@ -215,6 +227,32 @@ export async function __updateCommunity(
   if (directory_page_chain_node_id !== undefined) {
     chain.directory_page_chain_node_id = directory_page_chain_node_id;
   }
+  if (namespace !== undefined) {
+    if (!transactionHash) {
+      throw new AppError(Errors.InvalidTransactionHash);
+    }
+
+    if (!ownerOfChain) {
```

Comment:
Why not move the filter check down here, to avoid having to use the `let` pattern and keeping the variable around for the entire function when it's only used here?

Reply:
Basically we will need some let pattern unless we perform a second db call in order to get the user's addresses. This is because we already do it once above (see line 68). I use this let pattern to avoid calling the db twice for the same data.